controllers/flashsystemcluster_controller.go

```diff
@@ -345,11 +325,35 @@ func (r *FlashSystemClusterReconciler) reconcile(instance *odfv1alpha1.FlashSyst
 		instance.Status.Phase = util.PhaseProgressing
 	} else {
 		instance.Status.Phase = util.PhaseNotReady
-	}
+	} // vered : TODO why not to return error in case reconcile is not complete?
 
```

Comment:
@tyichye19  - what do you think?

Reply:
Decided to test failure scenarios on a different ticket 
https://trello.com/c/F62llqJp/161-test-driver-failure-what-if-1-manager-is-up-and-2nd-is-down-whats-the-fsc-state